### PR TITLE
Tweak comparison sentence

### PR DESCRIPTION
**Pull request by AttwellBrian**

### CODE REVIEW

**Comment by AttwellBrian** [comment]
Thought this was more accurate. But feel free to reject if you think it is too wordy.

File: README.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ To read about the backstory on why we created RIBs, see [this blog post](https:/
 
 #### What is the difference between RIBs and MV*/VIPER?
 
-MVC, MVP, MVVM and VIPER are architecture patterns. RIBs is a framework. What differentiates RIBs from these patterns is these:
+MVC, MVP, MVVM, MVI and VIPER are architecture patterns. RIBs is a framework. What differentiates RIBs from frameworks based on MV*/VIPER is:
```
Comment (sbarow) [inline README.md]: Small nit, can we move `MVI` before `MVVM`

Comment (AttwellBrian) [reply README.md]: Sure.
